update project profile of home unite us page

trishajjohnson opened the pull request:
Fixes #3084

### What changes did you make and why did you make them ?

  -Removed Abiha Ali from leadership roster
  -Added Ben Ross to leadership roster with their name, github handle, role and slack id.


### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
<!-- Note, if your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images -->
<!-- If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag -->

<details>
<summary>Visuals before changes are applied</summary>

![image](https://user-images.githubusercontent.com/66317088/169482158-f2584294-53a0-4f19-a8b5-195fd308d7d3.png)

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
![image](https://user-images.githubusercontent.com/66317088/169482551-a99bf552-31bd-4dda-be3a-00aa3b23fa24.png)

</details>

trishajjohnson committed:
removed Abiha Ali and added Ben Ross under leadership for home unite us page

Jaretzbalba commented:
ETA: End of the day on 5/22
Availability: 2 hours

Jaretzbalba commented:
Hey @trishajjohnson, can you make sure to add this pr to the project board?

Jaretzbalba reviewed (changes requested):
@trishajjohnson Thank you for working on this issue! Everything looks good but for consistency among the project pages, could you please change the roles from "Product Management" to "Product Manager" for Regan, Kristy, and Ben?

answebdev commented:
ETA: 5/20/2022 (end of day)
Availability: 2 hours

answebdev reviewed (changes requested):
@trishajjohnson Thank you for submitting your PR. Good job removing Abiha Ali from the leadership roster and adding Ben Ross to the leadership roster. I was able to see this change locally, as well. However, the link for Ben Ross's picture used in line 51 of `_projects/home-unite-us.md` is the same link used for Abiha Ali, so this needs to be changed so that the link for Ben Ross's picture is used.

I also second what @Jaretzbalba said about changing the roles from "Product Management" to "Product Manager" for Regan, Kristy, and Ben. That way, it will be consistent with the other project pages, as in the following examples:

<details>
  <summary>Food Oasis Project Page</summary>

![food_oasis](https://user-images.githubusercontent.com/36783010/169621010-82933b8a-a6b8-44f8-83af-e30b70fb0964.jpg)

</details>

<details>
  <summary>Brigade Organizers Playbook Project Page</summary>

![brigade_organizers_playbook](https://user-images.githubusercontent.com/36783010/169621028-d1ca7861-b124-44fb-8184-36e34c00de40.jpg)

</details>

<details>
  <summary>100 Automations Project Page</summary>

![100_automations](https://user-images.githubusercontent.com/36783010/169621089-54e5c39e-9aff-4b9b-9571-2ab71ff3dbcd.jpg)

</details>

And also, yes, make sure to add this PR to the project board. Thank you.

blulady reviewed (dismissed):
This pr is on the correct branch. There is a correct linked issue. The pr addresses the linked issue, namely to remove Abiha Ali from the Home Unit Us project info page and add Ben Ross. The pictures of visual changes look good. The links work in browser. The picture for Ben Ross looks like it is still Abiha. Make that Change and I think you are good to go.

trishajjohnson commented:
@answebdev and @Jaretzbalba I made the changed all Product Management roles to Product Manager, but I am unsure about the picture for Ben.  In the actual issue, there was no pic given to use for Ben.  I assumed that the pic for Abiha was a generic default pic, which is why I put it there.  Did I miss the url for Ben completely or is it not there in the issue?

UPDATE:

I just realized that the pictures used on the site are their github profile pictures, so I'll get Ben's from his github.

answebdev commented:
@trishajjohnson Thank you for making the change from "Product Management" to "Product Manager". But, were you able to see this change locally? I am not able to see this change. As you can see, I still see "Product Management", unless it's something on my end (see below). @Jaretzbalba Are you able to see the change to "Product Manager" locally on your end? Also, @trishajjohnson Were you able to get the link for Ben's GitHub picture?

<details>
  <summary>Home Unite Us - local</summary>

![home_unite_us](https://user-images.githubusercontent.com/36783010/170400327-51b0a218-edab-49fa-b382-48b75b8e3411.jpg)

</details>

trishajjohnson committed:
commiting revisions requested by PR reviewers

trishajjohnson commented:
@answebdev Sorry for the delay.  I hadn't pushed the revisions.  They are there now, so you should be able to see the changes.  I changed Product Management roles to Product Manager and I replaced Ben's picture with the correct url.  Ready to re-review.

answebdev reviewed (dismissed):
@trishajjohnson Thank you for pushing your revisions. I had a feeling my not being able to see your changes may have had something to do with that. But now I am able to see the changes. I can now see the change from "Product Management" to "Product Manager" for Regan, Kristy, and Ben, and I am now able to see Ben's GitHub picture as well. Great job!

kathrynsilvaconway commented:
ETA: End of Day on 5/30
Availability: 1 hour

trishajjohnson commented:
@JessicaLucindaCheng is this ready for merging, or do I need to request for another re-review?

JessicaLucindaCheng commented:
@hackforla/website-merge Please respond to @trishajjohnson: 
> is this ready for merging, or do I need to request for another re-review?

JessicaLucindaCheng commented:
@trishajjohnson Please reach out to Tamara or one of the merge team members. You can find their Slack info here: https://github.com/hackforla/website/wiki/Meet-the-Team

Jaretzbalba commented on _projects/home-unite-us.md:
```diff
@@ -49,7 +43,12 @@ leadership:
+    links:
+      slack: "https://hackforla.slack.com/team/U03BZ5WL8GY"
+      github: "https://github.com/rossbb"
+    picture: https://avatars.githubusercontent.com/u/5248025?v=4
```
@trishajjohnson To keep the code base easy to understand please change line 51 to https://avatars.githubusercontent.com/rossbb

trishajjohnson replied:
Hey @Jaretzbalba I went ahead and made that change you requested for the picture url for Ben Ross.  Would you be so kind as to re-review and approve for me?  Thanks!

Jaretzbalba reviewed (dismissed):
@trishajjohnson I see the changes made to the roles to keep everything consistent and it looks great! I just left a comment to fix a line concerning the linked image and everything should be good. Thank you again for working on this!

trishajjohnson committed:
fixed line 51 of home-unite-us.md updated url for Ben Ross' profile picture

trishajjohnson committed:
fixed picture link for Ben Ross

tamara-snyder reviewed (approved):
Looks great! The links work and Ben's picture is showing up. Thank you!
